client/ayon_fusion/plugins/create/create_saver.py

```diff
@@ -18,12 +18,12 @@ class CreateSaver(GenericCreateSaver):
     label = "Render (saver)"
     name = "render"
     product_type = "render"
-    description = "Fusion Saver to generate image sequence"
+    description = "Fusion Saver to generate a multi-file image sequence"
 
     default_frame_range_option = "current_folder"
 
     def get_detail_description(self):
-        return """Fusion Saver to generate image sequence.
+        return """Fusion Saver to generate a multi-file image sequence.
```

Comment:
I like this less personally - it's overly specific but also untrue. I'm pretty sure the `render` product type is completely fine to be a single frame. I do like the adjustment on the description of the image saver though.

@Sponge96 can you revert this one?

Reply:
Or actually - are you ok with reverting it? if so, let me push something to this branch too.

Reply:
I'll see how strong people feel on my end, I've reverted for now